src/styles/blocks/calendar.scss

```
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendarWidth;
+  
+  @for $i from  1 through 31 {
+    & > :nth-child(#{$i})::before {
+      content: "#{$i}";
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length--#{$i} > :nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+
+  @each $name, $margin in $weekDay {
+    &--start-day--#{$name} > :first-child {
+      margin-left: $margin;
+    }
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    margin-bottom: 1px;
+    width: 100px;
+    height: 100px;
+    background-color: #eee;
+    border: 1px black solid;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    transition: transform 0.5s;
```

create a variable for transition duration
```suggestion
    transition: transform $transitionDuration;
```